draft-ietf-tls-dtls-connection-id.md

```diff
@@ -345,6 +345,33 @@ data the following modification is made to the additional data calculation.
                       length_of_DTLSInnerPlaintext;
 ~~~
 
+# Peer Address Update {#peer-address-update}
+
+When a datagram with a CID is received that has a source address different than the one currently associated with the DTLS connection, the receiver MUST NOT replace the source address currently stored in the association with the source address specified in the received datagram unless the following conditions are met:
```

Comment:
Rewrapping this only.
```suggestion
When a datagram with a CID is received that has a source address different than
the one currently associated with the DTLS connection, the receiver MUST NOT
replace the source address currently stored in the association with the source
address specified in the received datagram unless the following conditions are
met:
```

However, I would also suggest that when you talk about "source address" (other than the first instance), you instead use "peer address", perhaps as follows:

```suggestion
When a datagram with a CID is received that has a source address different than
the one currently associated with the DTLS connection, the receiver MUST NOT
replace the address it uses for sending datagrams to its peer with the source
address specified in the received datagram unless the following conditions are
met:
```

Reply:
I am failing to see the changes in the proposed text. I understand why you want to use the term 'peer address' in context and I proposed some text